### PR TITLE
Do not disable any logger

### LINKED ISSUE #386: logs from switch.py are not showing
after the update about the logs https://github.com/kytos/kytos/pull/377, some logs are not showing...

### PR DESCRIPTION
**Pull request by cemsbr**
Fix #386

List of loggers that *were* disabled:
- concurrent.futures logger
- kytos.core.buffers logger
- kytos.core.napps.manager logger
- kytos.core.switch logger
- kytos.core.tcp_server logger
- socketio logger

### CODE REVIEW

File: kytos/core/logs.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def load_logging_file(cls, logging_file):
         cls.configuration.read(logging_file)
 
         try:
-            config.fileConfig(logging_file)
+            config.fileConfig(logging_file, disable_existing_loggers=False)
         except FileNotFoundError:
             cls.configuration.set('handler_syslog', 'args', '[]')
             config.fileConfig(cls.configuration)
```
Comment (beraldoleal) [inline kytos/core/logs.py]: Should not the line be here too?

**Review by beraldoleal — CHANGES_REQUESTED** [review]

File: kytos/core/logs.py (hunk 1 above)
Comment (cemsbr) [reply kytos/core/logs.py]: good catch. I refactored not to repeat code.

**Review by diraol — APPROVED** [review]

File: kytos/core/logs.py [hunk 2]
```diff
@@ -3,34 +3,32 @@
...
-           logging_file(str): Address of logging configuration file.
+           config_file (str, Path): Configuration file path.
         """
-        cls.configuration = RawConfigParser()
-        cls.configuration.read(logging_file)
-
-        try:
-            config.fileConfig(logging_file)
-        except FileNotFoundError:
-            cls.configuration.set('handler_syslog', 'args', '[]')
```
Comment (macartur) [inline kytos/core/logs.py]: I guess you forgot to verify the handler_syslog. This line change the 'handler_syslog' if the file /dev/syslog don't exists.

**Review by macartur — CHANGES_REQUESTED** [review]

**Comment by cemsbr** [comment]
@beraldoleal and @macartur, I believe all the issues are solved now. There's also tests for this issues and 100% coverage of logs module.

**Pull request merged**